framework/transport/ieee1905_transport/test/ieee1905_transport_test.cpp

```diff
@@ -388,15 +388,15 @@ int main(int argc, char *argv[])
         if (!ieee1905_bridge_ifname.empty()) {
             using Flags = InterfaceConfigurationRequestMessage::Flags;
             strncpy(interface_configuration_request_msg.metadata()->interfaces[n].ifname,
-                    ieee1905_bridge_ifname.c_str(), IF_NAMESIZE);
+                    ieee1905_bridge_ifname.c_str(), IF_NAMESIZE - 1);
```

Comment:
This seems to be an unrelated change...

If you're doing this, use copy_string like everywhere else. Because now it's actually still not correct since the terminating '\0' is not added.

Reply:
Changed to use copy_string.